### PR TITLE
added readme in pt_BR

### PR DESCRIPTION
**Pull request by royopa**

### CODE REVIEW

File: README-PT-BR.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+Propondo uma Recomendação de Padrão
+-----------------------------------
+
+Para propor uma recomendação de padrão (PSR):
+
+- faça um fork desse repositório, crie um branch, faça checkout desse branch,
+adicione a PSR em `proposed/`, faça o push do branch para o Github e envie
+um pull request; ou,
+
+- crie um ticket para iniciar uma discussão no Github; or,
```
Comment (pauloelr) [inline README-PT-BR.md]: missing translation of "or"

**Comment by royopa** [comment]
@pauloelr corrected. Thanks!

**Comment by pauloelr** [comment]
:+1: 
If it's woth for something i did a revision on the text and i'ts ok.

**Comment by royopa** [comment]
A added the PSR-3 - Translation to pt_BR to PR

**Comment by henriquemoody** [comment]
This is duplicated of #402 in some part and another pull request (#553) was created after this one to translate PSR-3, which I did review.

**Pull request closed without merging**